lib/auth/webauthnwin/api.go

```diff
@@ -158,6 +161,19 @@ func Register(
 	}, nil
 }
 
+var (
+	// promptPlatformMessage is the message shown before Touch ID prompts.
+	promptPlatformMessage = "Using platform authenticator, follow the OS dialogs"
+	// promptWriter is the writer used for prompt messages.
+	promptWriter io.Writer = os.Stderr
```

Comment:
Make it public so other packages could override/customize?

```suggestion
	// PromptPlatformMessage is the message shown before Touch ID prompts.
	PromptPlatformMessage = "Using platform authenticator, follow the OS dialogs"
	// PromptWriter is the writer used for prompt messages.
	PromptWriter io.Writer = os.Stderr
```